doc/news/changes/minor/20170725DiIlioEtAl

```diff
@@ -0,0 +1,2 @@
+Fixed: MappingQGeneric::transform_real_to_unit_cell had a bug where, for certain parallelograms, a problem with subtracting nearly equal floating point numbers ruined most of the digits of accuracy in the coordinate transformation.
+<br> (Giovanni Di Ilio, Jean-Paul Pelteret, David Wells, 2017/07/25)
```

Comment:
Your name should be ahead of mine - I've merely acted as middle management on this one :-)